### PR TITLE
Accept session as argument enhancement

### PR DESCRIPTION
**Pull request by martinavagyan**
We used similar approach to Zeeguu API: using decorators for checking session number
Example: https://zeeguu.herokuapp.com/?sessionID=11010001

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -1,6 +1,11 @@
 <!DOCTYPE html>
 <html>
 	<head>
+		<!-- Current session -->
+		<script>
+			var sessionID =  {{ sessionID }};
```
Comment (mircealungu) [inline templates/index.html]: why is this needed? we should avoid placing the session
number in the source code of the page if possible:
in theory this is a security issue since any extension 
can access all the source code of a page. 

is there an alternative?

Comment (martinavagyan) [inline templates/index.html]: I dislike this solution as well, but this was a quick fix to the problem also I saw some other places in zeeguu platform using it. 
Besides since the bookmarks are loaded in front end, the session number will be visible to outsides with the current implementation.

Comment (mircealungu) [inline templates/index.html]: Ok. Let's deploy it for now like this for testing, and we think of a better design after your exams are over and you can focus fully on making the practice as a service great again!!

Comment (martinavagyan) [inline templates/index.html]: Agreed 👍

Comment (mircealungu) [inline templates/index.html]: So that we don't forget, please add it as a new issue. Unless you just did that :)
